frontend/src/pages/restartPage/connectivityWarningDialog/ConnectivityWarningDialog.tsx

```diff
@@ -25,10 +25,10 @@ export const getContent = (shouldMoveAwayFromAp: boolean) => {
   if (shouldMoveAwayFromAp) {
     return (
       <>
-        Your pi-top is now connected to the Wi-Fi network you configured in a previous step, but your computer is still on the 'pi-top-XXXX' network.
+        Your pi-top was connected to a Wi-Fi network in a previous step, but your computer is still on the pi-top's hotspot.
```

Comment:
I think this could be shortened further:
'You connected your pi-top to Wi-Fi earlier, but your computer is still on the pi-top's WiFi hotspot.